zcash_primitives/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ and this library adheres to Rust's notion of
 ## [0.17.0] - 2024-08-26
 
 ### Changed
-- Update dependencies to `zcash_protocol 0.3.0`, `zcash_address 0.5.0`
+- Update dependencies to `incrementalmerkletree 0.7`, `orchard 0.10`,
+  `sapling-crypto 0.3`, `zcash_protocol 0.3.0`, `zcash_address 0.5.0`.
```

Comment:
I edited the wrong part of the changelog here and it wasn't caught in review. I'll fix this in the release train PR.